stable_baselines/sac/sac.py

```diff
@@ -389,9 +390,13 @@ def learn(self, total_timesteps, callback=None, seed=None, log_interval=4, tb_lo
                     infos_values = []
             return self
 
-    def action_probability(self, observation, state=None, mask=None):
+    def action_probability(self, observation, state=None, mask=None, actions=None):
+        if actions is not None:
+            raise ValueError("Error: DDPG does not have action probabilities.")
```

Comment:
two things:
1. typo, that is SAC, not DDPG
2. in theory, because SAC is using a gaussian policy, you could extract the probability. In practice, it uses a squashing function `tanh` that makes it difficult. One idea would be to compute the probability for `tanh^{-1}(action)`.